zebracrossing/campaigns/views.py

```diff
@@ -119,3 +172,39 @@ def download_item(request, item):
     response = HttpResponse(item, content_type=mimetype)
     response["Content-Disposition"] = f'filename="{item.name}"'
     return response
+
+
+def create_date_from_table_header(date):
+    result = date[str(date).find("(") + 1 : str(date).find(")")]
+    day = result.split("/")[0]
+    month = result.split("/")[1]
+    year = str(datetime.today().year)
+    date_object = datetime.strptime(f"{year}-{month}-{day}", "%Y-%m-%d").date()
+    return date_object
+
+
+def save_to_table(request):
+    if request.method != "POST":
+        return HttpResponseNotAllowed(["GET"])
```

Comment:
The argument is the allowed response, so that should return `HttpResponseNotAllowed(["POST"])`